sql/core/src/main/scala/org/apache/spark/sql/execution/command/functions.scala

```diff
@@ -157,6 +157,9 @@ case class DropFunction(
         throw new AnalysisException(s"Specifying a database in DROP TEMPORARY FUNCTION " +
           s"is not allowed: '${databaseName.get}'")
       }
+      if (FunctionRegistry.builtin.functionExists(functionName)) {
```

Comment:
Thanks for the investigation! I guess we won't support this feature in 2.0, but need @yhuai to confirm.